.github/workflows/lint-format-commitlint.yml

```
@@ -0,0 +1,38 @@
+
+      - name: Run ESLint
+        run: pnpm exec eslint .
+
+      - name: Run Prettier (check mode)
+        run: pnpm exec prettier --check .
+
+      - name: Run Commitlint on PR title and commits
+        uses: wagoid/commitlint-github-action@v5
+        with:
+          configFile: commitlint.config.cjs
+          checkCommits: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)